### DISCUSSION TITLE
TV3 Mini [LV]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#902] [Q&A]
### Channel ID

TV3Mini.lv

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by DJ1999-max** [comment]
Please check stream link.

**Comment by dantheman121** [comment]
> Please check stream link.

its blocked 
i got a 403 error when trying to watch it in vlc

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19866
